packages/venia-ui/lib/components/ProductSort/productSort.js

```diff
@@ -113,6 +113,12 @@ ProductSort.propTypes = {
 
 ProductSort.defaultProps = {
     availableSortMethods: [
+        {
+            text: 'Position',
+            id: 'sortItem.position',
+            attribute: 'position',
+            sortDirection: 'ASC'
+        },
```

Comment:
One thought I just had was to rewrite the `useSort` hook to return both `current` and `available` sort rather than just be a state holder for current sort. Right now we have this odd coupling of sort methods with this array in `ProductSort` (which has a prop, so it can be passed the details from the parent).